samcli/commands/_utils/options.py

```diff
@@ -789,6 +789,16 @@ def use_container_build_option(f):
     return use_container_build_click_option()(f)
 
 
+def plan_file_click_option():
+    return click.option(
+        "--plan-file", type=click.Path(), help="Used for passing a custom plan file when executing the Terraform hook."
```

Comment:
do we need to make the parameter name more specific to terraform like `--terraform-plan-file` ?